cohortextractor2/dsl.py

```diff
@@ -158,3 +160,10 @@ class PatientSeries:
 
     def __init__(self, value: Value):
         self.value = value
+
+
+Expression = Union[PatientSeries, bool, int, str]
```

Comment:
Would `Expression = PatientSeries | bool | int | str` work here? And maybe a better question… would it be more readable??

What's an Expression in this case? Could we add a short comment to make it more obvious?

Reply:
This gives me an error:

```
cohortextractor2/dsl.py:163: error: Unsupported left operand type for | ("Type[PatientSeries]")  [operator]
cohortextractor2/dsl.py:166: error: Variable "cohortextractor2.dsl.Expression" is not valid as a type  [valid-type]
cohortextractor2/dsl.py:166: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases
```

Reply:
Ooh, pipe unions are 3.10, that's probably it?